src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((styles) => styles.trim())
+    .filter((styles) => styles !== '')
+    .reduce((cssStyles, styles) => {
+      const [key, value] = styles.split(':').map((item) => item.trim());
```

Comment:
The destructuring of `styles.split(':')` assumes that there will always be only one colon in the style declaration. If a style value contains a colon (e.g., in a pseudo-element or a base64 encoded image), this will lead to incorrect results. To ensure only the first colon is used to split the key and value, consider using `split(':', 2)`.